packages/nimble-components/docs/component-status.mdx

```diff
@@ -0,0 +1,16 @@
+import { Story, Meta } from '@storybook/addon-docs';
+import { componentStatus } from './component-status.stories';
+
+<Meta title="Components/Status Table" />
+
+# Component status
+
+Here are components that are either implemented or on the roadmap for Nimble. Something missing? [File a request](https://github.com/ni/nimble/issues/new/choose)!
+
+## Legend
+
+-   ✅ means the component is ready for general use
+-   ⚠️ is used for components that are "[incubating](/docs/incubating-docs--docs)". These are not ready for general use and may make breaking changes without a "major" semantic version bump.
+-   ⭕ means the component doesn't exist yet. Please submit requests by commenting on the linked issue.
+
+<Story of={componentStatus} />
```

Comment:
How about putting the legend in a markdown table, and moving it to the bottom? It's fairly self-explanatory, so likely doesn't need to be in the most prominent position on the page…

```suggestion
<Story of={componentStatus} />

## Legend

| Icon | Description                                                                                                                                                              |
| ---- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
| ✅   | The component is ready for general use.                                                                                                                                  |
| ⚠️   | The component is "[incubating](/docs/incubating-docs--docs)". These are not ready for general use and may make breaking changes without a "major" semantic version bump. |
| ⭕   | The component doesn't exist yet. Please submit requests by commenting on the linked issue.                                                                               |

```

Reply:
For some reason Storybook rendered the "Legend" header with no padding below the bottom of the table so I had to [add a nbsp](https://stackoverflow.com/questions/20543454/create-two-blank-lines-in-markdown). But once that's there, it's a nice improvement!